src/components/PostEditor/PostEditor.js

```diff
@@ -249,7 +249,7 @@ class PostEditor extends React.Component {
 
   handleTitleChange = (event) => {
     const title = event.target.value
-    title.length >= MAX_TITLE_LENGTH
+    title.length > MAX_TITLE_LENGTH
```

Comment:
so we actually do want a warning to appear when you have reached maximum length. otherwise there would be no way to know you had reached the end. the current error is not great and could definitely be improved by making it a less intense notice that you have hit the max length, but also that would increase the scope of this ticket. For now I think we should leave the way it is

Reply:
tbh, it seems this test can't fail normally because the html input control is limited at `MAX_TITLE_LENGTH` -- I think the only way around it would be a hack.

Reply:
im confused, I still think we want this to be >= so the error/warning message appears once you've reached the maximum. isn't this the place that does that?

Reply:
Ahhhhh I see -- yeah, that's not an error but I understand that it needs to pretend it's an error in order to let the user know they hit the max. Hrm. I agree, out of scope, let's open a new ticket to soften that error. I'll undo this change and revise the test and push.

Reply:
Okay, code and test updated